torch/utils/data/dataloader.py

```diff
@@ -1129,16 +1131,22 @@ def __init__(self, loader):
                 ),
             )
             w.daemon = True
-            # NB: Process.start() actually take some time as it needs to
```

Comment:
This comment goes back to #11432, but since then a lot of [edge case handling](https://github.com/pytorch/pytorch/blob/main/torch/utils/data/dataloader.py#L1522) has been added for worker shutdown so I don't think this is an issue.